src/FNA3D_Driver_Vulkan.c

```diff
@@ -187,9 +188,13 @@ struct RenderPassHashMap
 typedef struct FramebufferHash
 {
 	VkImageView colorAttachmentViewOne;
+	VkImageView colorMultiSampleAttachmentViewOne;
```

Comment:
I'm pretty sure you could make colorAttachments and colorMultiSampleAttachments into fixed-length arrays, since it should get packed the same anyway. Might be a bit easier than wrangling 8 different struct members. :)

Reply:
I was playing it safe because I didn't know if the data would get packed correctly in an array, thanks for the heads up!